server/endpoint/health/searcher/endpoint_test.go

```diff
@@ -0,0 +1,140 @@
+package searcher
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/health-service/service/health"
+	"github.com/giantswarm/micrologger/microloggertest"
+	"github.com/giantswarm/operatorkit/client/k8srestconfig"
+	"github.com/spf13/viper"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/giantswarm/health-service/flag"
+	"github.com/giantswarm/health-service/server/middleware"
+	"github.com/giantswarm/health-service/service"
+)
+
+var testCases = []struct {
```

Comment:
I would like to have this more aligned with `fmt`.
This structure looks too unique to me in general - can we make it fit better?

https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md#table-driven-tests

Reply:
Ah, I hadn't seen those docs. I'll get this cleaned up.

Reply:
This should be fixed in my latest commit. Thanks for telling me about the `fmt` docs regarding unit tests.